content/authors/benjamin-macadam/_index.md

```diff
@@ -0,0 +1,59 @@
+# Display name
```

Comment:
The header section needs to start with a line with `---` in it at the top, Can you add that as the first line and hit commit? That should update the PR and then i think this will work